modules/ROOT/pages/advtable.adoc

```diff
@@ -63,6 +63,47 @@ A numeric and alpha value series are available by default. The available value s
 
 image::advtable-row-numbering.png[Table with numeric row numbering column and row numbering menu open (Numeric item checked)]
 
+== Excluding header and footer rows from row numbering
+
+A commonly desired customisation of the row-numbering column is that a table’s header and footer rows not be included in the numbering.
+
```

Comment:
This content should go with the existing examples in the `advtable_value_series` partial which is already included on this page. `include::partial$configuration/advtable_value_series.adoc`

Reply:
> This content should go with the existing examples in the `advtable_value_series` partial which is already included on this page. `include::partial$configuration/advtable_value_series.adoc`

Edited to basically follow this. The ‘commonly desired’ sentences, including the `xref` are  still at the end of the *Adding row numbering to a table*. The example proper, however, is now in the *advtable_value_series* section.